release_notes/ocp-4-10-release-notes.adoc

```diff
@@ -175,6 +175,29 @@ For more information, see xref:../updating/understanding-upgrade-channels-releas
 [id="ocp-4-10-web-console"]
 === Web console
 
+[id="ocp-4-10-web-console-developer-perspective"]
+==== Developer perspective
+* With this update, you can specify the name of a service binding connector in the *Topology* view while making a binding connection.
+* With this update, creating pipelines workflow has now been enhanced:
+** You can see when the pipeline has been created but not started by looking at the status of the pod in the *Topology* view.
+** You can create an application using the *Import from Git* workflow, along with a template-generated pipeline by selecting the *Add pipeline* option. The webhook resources and URL are visible in the side panel of the selected resources.
+** You can opt out of the default Tekton Hub integration by setting the parameter `enable-devconsole-integration` to `false` in the `TektonConfig` custom resource. When you opt out, only the platform compatible cluster tasks for your namespace are listed in the *Pipeline builder* -> *Add tasks* view.
```

Comment:
I Suggest the following changes, Please update the text as you see fit.

```suggestion
** Pipeline builder will contain the Tekton Hub tasks that are supported by the cluster's Architecture, all other unsupported tasks are excluded from the list.
** You can opt out of the default Tekton Hub integration by setting the parameter `enable-devconsole-integration` to `false` in the `TektonConfig` custom resource.
```

Reply:
Thanks for the suggestion @karthikjeeyar I'll incorporate these changes in the next edit.